activerecord/test/cases/adapters/postgresql/connection_test.rb

```diff
@@ -26,21 +26,15 @@ def teardown
     end
 
     def test_encoding
-      assert_queries(1) do
```

Comment:
Can you keep the `assert_queries`?
Before #29253, `@connection.encoding` executed two queries, these assertions will prevent to lose that silently.

Reply:
Thanks for the information. Restored `assert_queries`.